### PR TITLE
TRIP-6504: Add Sentry tracking configuration

### PR DESCRIPTION
**Pull request by tourlane-a-lanovyy**
## Context/Change

Add Sentry configurations for error tracking.
Provided dynamic ENV variables to configure Sentry project.

### Results 
image
image

## Reference
[TRIP-6504](https://tourlane.atlassian.net/browse/TRIP-6504)

### CODE REVIEW

**Comment by tourlane-a-lanovyy** [comment]
**Related:** We should configure **env** variables and `.sentryclirc` file on the deployment env.

Check `.env.example` for `NEXT_PUBLIC_SENTRY_DSN`

And we should add `apps/web/.sentryclirc` 
```
[auth]
token=#####
```

File: apps/web/sentry.client.config.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import * as Sentry from "@sentry/nextjs";
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn: SENTRY_DSN,
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0,
```
Comment (haffla) [inline apps/web/sentry.client.config.ts:9]: Let's set this to 0. We don't want to measure performance of the client.

File: apps/web/sentry.server.config.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as Sentry from "@sentry/nextjs";
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn: SENTRY_DSN,
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0,
```
Comment (haffla) [inline apps/web/sentry.server.config.ts:9]: Same here actually. Let's disable it. We just want to capture errors.

File: apps/web/sentry.client.config.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  // ...
+  // Note: if you want to override the automatic release value, do not set a
+  // `release` value here - use the environment variable `SENTRY_RELEASE`, so
+  // that it will also get attached to your source maps
```
Comment (haffla) [inline apps/web/sentry.client.config.ts:13]: We will also need to set the `environment` probably via environment variable. Something like `NEXT_PUBLIC_SENTRY_ENV`.

Same for the server config.

File: apps/web/sentry.client.config.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import * as Sentry from "@sentry/nextjs";
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn: SENTRY_DSN,
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0,
+  // ...
```
Comment (haffla) [inline apps/web/sentry.client.config.ts:10]: We will also need to set the `environment` probably via environment variable. Something like `NEXT_PUBLIC_SENTRY_ENV`.

Same for the server config.

File: apps/web/sentry.properties [hunk 5]
```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
```
Comment (haffla) [inline apps/web/sentry.properties:1]: I was really hoping that we do not need this file. Have you tried it without it? Usually the DSN uniquely identifies the org and project in Sentry.

**Pull request closed without merging**